src/stream/stream.rs

```diff
@@ -273,3 +330,50 @@ where
         }
     }
 }
+
+#[derive(Debug)]
+pub struct AnyFuture<'a, S, F, T>
```

Comment:
i'm bad at naming things xD
```rust
type AnyFuture = Future<Output = Any>
```
sorry :joy_cat:

Reply:
I'm thinking we should probably hide this Future using the `impl/ImplFuture` macro

Reply:
is there any example, where is that macro?

Reply:
On a phone right now, but look for `macro_rules` in `Stream`. There's a macro that returns `ImplFuture` defined.

Reply:
ok, I will try to do so.